admin_guide/install/deploy_router.adoc

```diff
@@ -241,6 +242,58 @@ To remove a passphrase from a keyfile, you can run:
 # openssl rsa -in <passwordProtectedKey.key> -out <new.key>
 ----
 
+[[using-the-container-network-stack]]
+
+=== Using the Container Network Stack
+
+The OpenShift router runs inside a Docker container and the default behavior is
+to use the network stack of the host (i.e., the node where the router container
+runs). This default behavior has performance benefits as the network traffic
+from remote clients no longer needs to go through a double hop through user
+space to reach the target service and container.
```

Comment:
Changed to:

```
This default behavior benefits performance because network traffic from remote clients does not need to take multiple hops through user space to reach the target service and container.
```

Also synced up the similar line in the upgrade topic.